subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1658,6 +1658,7 @@ static void light_ctrl_srv_reset(struct bt_mesh_model *model)
 
 	ctrl_disable(srv);
 	net_buf_simple_reset(srv->pub.msg);
+	srv->resume = CONFIG_BT_MESH_LIGHT_CTRL_SRV_RESUME_DELAY;
```

Comment:
Shouldn't it be stored in the persistent memory instead? and restored after reset

Reply:
Since we are resetting the mesh node or unprovision the node, this could mean that the user wants to relocate the node or even use it in other site and the older configuration could be different. 
To be generic, I think it would be better restore to default value. Provisioner will always know the start value in the provisioning.
When the node is provisioned again, user can change it back to the desired value.

Reply:
Hi @alxelax , this commit will do the same that is already done for bt_mesh_light_ctrl_srv_cfg  and bt_mesh_light_ctrl_srv_reg_cfg  a few lines before ( https://github.com/nrfconnect/sdk-nrf/blob/90f53087fe131804b99e4eb8391cd6327a2a0e12/subsys/bluetooth/mesh/light_ctrl_srv.c#L1649-L1656 ).